Readme Improvements (#446)
EddieHubCommunity/LinkFree: README.md

# LinkFree - EddieHub

> + LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript. 
> + It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).

Reviewer: ```suggestion
- LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript. 
- It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
```

![Screenshot](https://user-images.githubusercontent.com/60853067/133296120-dbdb1799-4cca-4708-81ce-05edc65e59c9.png)

> ## Note for Hacktoberfest

Reviewer: ```suggestion
## Note for Hacktoberfest

```

## 👨‍💻 Contributing

> + Contributions are what make the open source community such an amazing place to learn, inspire, and create. 
> + Any contributions you make are **greatly appreciated**.
> + Check out [Contributing.md](Contributing.md) file for more information.

Reviewer: ```suggestion
- Contributions are what makes the open source community such an amazing place to learn, inspire, and create. 
- Any contributions you make are **greatly appreciated**.
- Check out our [contribution guidelines](https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md) for more information.
```
